mangaki/mangaki/tests/test_error_pages.py

```diff
@@ -0,0 +1,31 @@
+from django.test import TestCase, Client
+from mangaki.models import Work, Category, Trope
+
+
+class ErrorPageTest(TestCase):
+
+    def setUp(self):
+        self.client = Client()
+        anime = Category.objects.get(slug='anime')
+        Work.objects.create(
+            title='Initial D',
+            source='Fujiwara Bunta',
+            category=anime
+        )
+        Trope.objects.create(
+            trope='INERTIA DORIFTU ?!',
+            author='Keisuke Takahashi',
+            origin=Work.objects.get(title='Initial D')
+        )
+        self.work = Work.objects.get(title='Initial D')
+        self.trope = Trope.objects.get()
+
+    def test_trope(self):
+        response = self.client.get('/404/')
```

Comment:
`/404/` déclenche bien une erreur `404` d'une façon détournée (cf `mangaki.models.WorkList.get_queryset`) suite à la route catch-all. Ceci dit, il s'agit d'une URL qui *pourrait* être valide - pourquoi ne pas plutôt utiliser quelque chose comme `/does/not/exists/` en vérifiant que [resolve](https://docs.djangoproject.com/en/1.10/ref/urlresolvers/#resolve) lance bien une exception `Resolver404` en safety check ?